tools/walletextension/wallet_extension.go

```diff
@@ -309,9 +309,10 @@ func (w *WalletExtension) UserExists(hexUserID string) bool {
 		return false
 	}
 
+	// Check if user exists and don't log error if user doesn't exist, because we expect this to happen in case of
+	// user revoking encryption token or using different testnet.
```

Comment:
```suggestion
	// Check if user exists and don't log error if user doesn't exist, because we expect this to happen in case of
	// user revoking encryption token or using different testnet.
	// todo add a counter here in the future
```